src/styles/layout/header.less

```diff
@@ -0,0 +1,105 @@
+/* -----------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+header.less
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+----------------------------------------------------------------------------- */
+
+
+
+/* -----------------------------------------------------------------------------
+Header
+----------------------------------------------------------------------------- */
+
+.header {
+
+  .clearfix();
+  position: relative;
+  width: 100%;
+  z-index: @z-index-header;
+  background: color-lighten(@neutral, -50);
```

Comment:
HUUUUUGGGGGE PAIN IN THE ASS AND OVERKILL.  Order by context rather than alphabetical makes a lot more sense.  done